src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -874,6 +874,8 @@ public static string MapToSnapshotNotSupportedWithoutContext(string snapshotType
          public static string SnapshotFileMismatch(string desiredType) => $"Snapshot file cannot be used to load a {desiredType.ToLowerInvariant()}.";
 
          public static string CannotLoadRelatedItemAsObjectAlreadyExistInProject(string objectType, string objectName) => $"Cannot load related item into project. A {objectType.ToLower()} named '{objectName}' already exists.";
+
+         public static string CompoundGroupNotFoundFor(string compoundGroup, string compoundName) => $"Cannot find compound group '{compoundGroup}' for compound '{compoundName}'";
```

Comment:
Unsure if those kind of error should be logged or throw exception. We should decide of the best strategy in those cases.
This happens if the alternative selected in the simulation does not exist in the compound. Typically editing the json by hand